bin/internal/goldens.version

```diff
@@ -1 +1 @@
-49f8198b72f6e12c65fe1db2e46162de0204e671
+38e117bcd79860a3f239872c579acac716e70d49
```

Comment:
This commit does not appear to exist in https://github.com/flutter/goldens/commits/master which is causing these golden file tests to fail.

Reply:
Ah sorry I haven't merged the golden PR yet. I changed the golden hash to prevent `flutter test` from resetting the head to `master` so I can test with my golden fork. I'll update it once the golden PR is merged.

Reply:
Updated.